src/Modules/SharedModules/components/DynamicHeader/DynamicHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export default function DynamicHeader({title,btn}) {
```

Comment:
We can change the name `btn` here to more meaningful name that indicate that it is a text not an actual button.